en/developer_guide/configuration.rst

```diff
@@ -0,0 +1,47 @@
+Server Configuration
+====================
+
+To know about server configuration see `requirement`_.
+
+Cron tasks
+----------
+
+Cron tasks are for schedule jobs (commands or shell scripts) to run periodically at fixed times, dates, or intervals.
+Cron tasks typically automates system maintenance.
+
+Cron tasks on Open Orchestra
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Open Orchestra has cron tasks which allow to execute commands regularly. Those tasks are launch every day at 2 am in the morning.
+
+* **Sitemap**: this task generates automatically the files sitemap.xml for every sites.
+* **Robots**: this task generates automatically the files robots.txt for every sites.
+
+For more information on the command which generates the sitemap.xml and the robots.txt see the documentation `sitemapAndRobotsFiles`_
+
+To modify the moment when every task will be executed, modify variables in ``provisioning/roles/orchestraCron/vars/main.yml``.
+
+Create a cron task
+~~~~~~~~~~~~~~~~~~
+
+To add a cron task create a file named like you task in the ansible role ``provisioning/roles/orchestraCron/tasks``.
+
+In your file use the ansible cron command:
+
+.. code-block:: yaml
+
+    cron: name="description de la tâche" min="minutes" hour="heure" day="day" month="month" job="tâche"
+
+For more precision on the command cron go to see the documentation `ansible cron module`_.
+
+Add your task in the file ``provisioning/roles/orchestraCron/tasks/main.yml``
+
+.. code-block:: yaml
+
+    - include: tache.yml
+
+The used variables are in ``provisioning/roles/orchestraCron/vars/main.yml ``.
```

Comment:
Note sure about the meaning. What about "The variables in use are in" or "The available variables"